### DISCUSSION TITLE
"Show unauthorized links" in tag listing

### DISCUSSION THREAD

**Discussion by stekky81** [joomla/joomla-cms#47158] [2. Cold Features]
### Steps to reproduce the issue
1) turn on "show unauthorized links" option on global Articles options 
2) create a new article, set access level to registered and add a "test" tag

### Expected result
The new article is visible (with a "register to read more" link):
- in category listing
- in mod_articles_* listing
- in "test" tag listing

### Actual result
- the new article is correctly shown both in category listing and in mod_articles_* with a "register to read more" link
- the new article does not appear in "test" tag listing

### System information (as much as possible)
Joomla! 3.9.20

### Additional comments
the new article should also appear in tag listing to have consistent behavior

**Comment by Quy** [comment]
Duplicate #21407 ?

**Comment by SharkyKZ** [comment]
It's more or less expected behavior. `Show unauthorised links` option is specific to `com_content`. Meanwhile `com_tags` supports many content types from different components.

**Comment by stekky81** [comment]
> Duplicate #21407 ?

No, it's a different issue.

**Comment by stekky81** [comment]
> It's more or less expected behavior. `Show unauthorised links` option is specific to `com_content`. Meanwhile `com_tags` supports many content types from different components.

I understand your point of view, but it could be the desired behavior even in third-party components. The most common use case is to enable a membership system for which the introtext is shown and for the "read more" it is necessary to register. Of course the option may or may not be enabled in each component. The main problem is that now for `com_content` the behavior is not consistent between all views.

**Comment by ghost** [comment]
related #24551
